

==> WARNING: A newer version of conda exists. <==
  current version: 4.12.0
  latest version: 23.7.3

Please update conda by running

    $ conda update -n base -c defaults conda



EnvironmentNotWritableError: The current user does not have write permissions to the target environment.
  environment location: /deepstore/software/anaconda3/2022.05
  uid: 17701115
  gid: 10384




==> WARNING: A newer version of conda exists. <==
  current version: 4.12.0
  latest version: 23.7.3

Please update conda by running

    $ conda update -n base -c defaults conda


2023-08-30 16:41:01.573259: E external/xla/xla/stream_executor/cuda/cuda_dnn.cc:413] There was an error before creating cudnn handle (302): cudaGetErrorName symbol not found. : cudaGetErrorString symbol not found.
Traceback (most recent call last):
  File "/local/232479/main.py", line 6, in <module>
    from include.init import ModelInitializer_2d
  File "/local/232479/include/init.py", line 4, in <module>
    from include.config import key_he, key_x_u, key_x_f, key_t_u, key_t_f
  File "/local/232479/include/config.py", line 3, in <module>
    key = jax.random.PRNGKey(42)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/random.py", line 177, in PRNGKey
    return _return_prng_keys(True, _key('PRNGKey', seed, impl))
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/random.py", line 139, in _key
    return prng.seed_with_impl(impl, seed)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 406, in seed_with_impl
    return random_seed(seed, impl=impl)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 689, in random_seed
    return random_seed_p.bind(seeds_arr, impl=impl)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 701, in random_seed_impl
    base_arr = random_seed_impl_base(seeds, impl=impl)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 706, in random_seed_impl_base
    return seed(seeds)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 935, in threefry_seed
    return _threefry_seed(seed)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/traceback_util.py", line 166, in reraise_with_filtered_traceback
    return fun(*args, **kwargs)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/pjit.py", line 253, in cache_miss
    outs, out_flat, out_tree, args_flat, jaxpr = _python_pjit_helper(
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/pjit.py", line 166, in _python_pjit_helper
    out_flat = pjit_p.bind(*args_flat, **params)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/core.py", line 2596, in bind
    return self.bind_with_trace(top_trace, args, params)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/core.py", line 389, in bind_with_trace
    out = trace.process_primitive(self, map(trace.full_raise, args), params)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/core.py", line 821, in process_primitive
    return primitive.impl(*tracers, **params)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/pjit.py", line 1209, in _pjit_call_impl
    return xc._xla.pjit(name, f, call_impl_cache_miss, [], [], donated_argnums,
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/pjit.py", line 1192, in call_impl_cache_miss
    out_flat, compiled = _pjit_call_impl_python(
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/pjit.py", line 1128, in _pjit_call_impl_python
    always_lower=False, lowering_platform=None).compile()
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 2206, in compile
    executable = UnloadedMeshExecutable.from_hlo(
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 2544, in from_hlo
    xla_executable, compile_options = _cached_compilation(
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 2454, in _cached_compilation
    xla_executable = dispatch.compile_or_get_cached(
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/dispatch.py", line 496, in compile_or_get_cached
    return backend_compile(backend, computation, compile_options,
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/profiler.py", line 314, in wrapper
    return func(*args, **kwargs)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/dispatch.py", line 464, in backend_compile
    return backend.compile(built_c, compile_options=options)
jax._src.traceback_util.UnfilteredStackTrace: jaxlib.xla_extension.XlaRuntimeError: FAILED_PRECONDITION: DNN library initialization failed. Look at the errors above for more details.

The stack trace below excludes JAX-internal frames.
The preceding is the original exception that occurred, unmodified.

--------------------

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/local/232479/main.py", line 6, in <module>
    from include.init import ModelInitializer_2d
  File "/local/232479/include/init.py", line 4, in <module>
    from include.config import key_he, key_x_u, key_x_f, key_t_u, key_t_f
  File "/local/232479/include/config.py", line 3, in <module>
    key = jax.random.PRNGKey(42)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/random.py", line 177, in PRNGKey
    return _return_prng_keys(True, _key('PRNGKey', seed, impl))
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/random.py", line 139, in _key
    return prng.seed_with_impl(impl, seed)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 406, in seed_with_impl
    return random_seed(seed, impl=impl)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 689, in random_seed
    return random_seed_p.bind(seeds_arr, impl=impl)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/core.py", line 386, in bind
    return self.bind_with_trace(find_top_trace(args), args, params)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/core.py", line 389, in bind_with_trace
    out = trace.process_primitive(self, map(trace.full_raise, args), params)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/core.py", line 821, in process_primitive
    return primitive.impl(*tracers, **params)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 701, in random_seed_impl
    base_arr = random_seed_impl_base(seeds, impl=impl)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 706, in random_seed_impl_base
    return seed(seeds)
  File "/home/yanw/.conda/envs/noise/lib/python3.10/site-packages/jax/_src/prng.py", line 935, in threefry_seed
    return _threefry_seed(seed)
jaxlib.xla_extension.XlaRuntimeError: FAILED_PRECONDITION: DNN library initialization failed. Look at the errors above for more details.
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[42556,1],0]
  Exit code:    1
--------------------------------------------------------------------------
cp: cannot stat '*.pdf': No such file or directory
cp: missing destination file operand after '*.png/home/yanw/uncertain/results/figures/0830'
Try 'cp --help' for more information.
cp: missing destination file operand after '/home/yanw/uncertain'
Try 'cp --help' for more information.
